WTWSMS/common/government_flavor/WtWSMS_government_flavor.txt

```diff
@@ -16,6 +16,9 @@ government_flavor = {
 	graphics = GFX_evt_zoroastrian_fire
 	priority = 2
 	trigger = {
+		NOT = {
+			has_landed_title = e_persia
+		}
```

Comment:
It may have been leftover from a previous commitment. In that form, Eranshar and Iranian Kingdom were two different Government Flavors.